README.md

```diff
@@ -1,40 +1,126 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio-Mobile-first-project
+<a name="readme-top"></a>
 
-> Web design for mobile version
-> A page that contain navigation bar and headline of portfolio page.
+<div align="center">
 
-## Built With
+  <img src="images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-- HTML, CSS
+  <h3><b>Capstone-project-one</b></h3>
 
-## General Info
+</div>
 
-This project is building a web for mobile. Here, the navigation bar and the headline is created.
-## Authors
+<!-- TABLE OF CONTENTS -->
 
-👤 Hiwot Fekadu Bayissa
+# 📗 Table of Contents
 
-- GitHub: [@hiwina](https://github.com/hiwina)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Clone](#clone)
+  - [Run](#run)
+- [👥 Authors](#authors)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
 
-## 🤝 Contributing
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone-project-one <a name="about-project"></a>
+
+> This is a Capstone Project for Microverse block one course.
+
+## 🛠 Built With <a name="built-with">HTML, CSS, & Javascript</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Responsive 
+- Dynamic
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is a link to deployed project.
+
+- [Live Demo Link](ttps://hiwina.github.io/Capstone-Project-1/)
```

Comment:
Kindly, I suggest you follow the previous reviewer's advice, the LIVE DEMO link, currently connects to your repository, and it should point to the deployed version of your site.